src/ceph-disk/ceph_disk/main.py

```diff
@@ -665,6 +665,19 @@ def get_dev_size(dev, size='megabytes'):
         os.close(fd)
 
 
+def is_diskdevice
```

Comment:
@dachary 
I'm open to suggestions for another function name. I though `is_blockdevice` but then that is exactly what it is not. It is the bare metal thingy ultimately, the diskdevice.